backend/src/main/java/com/messenger/backend/controllers/UserController.java

```diff
@@ -40,10 +40,16 @@ public Boolean getUserStatus(@RequestParam(value = "id") Integer id) {
         return userService.getUserStatus(id);
     }
 
-    @PutMapping(value = "/updateUser", consumes = MediaType.APPLICATION_JSON_VALUE) //update contact list
-
+    @GetMapping(value = "/createUser", consumes = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
In CRUD operations, "Create" is implemented by POST method. But, as I understand, using more simple method is acceptable. Especcially in our case, where the only parameter passed is newUserName (String)